app/controllers/idv/in_person/public/mock_address_search_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Idv
+  module InPerson
+    module Public
+      class MockAddressSearchController < ApplicationController
```

Comment:
My first thought was to add barebones versions of the actual controllers we want, eg `AddressSearchController` and `UspsLocationsController`, so that we can flesh them out over time

I'm fine with this approach too. Seems like a little more code turnover to later remove this 🤷